pom.xml

```diff
@@ -257,6 +260,17 @@
                 <artifactId>rxjava</artifactId>
                 <version>2.1.9</version>
             </dependency>
+            <dependency>
```

Comment:
do we need the reactive-streams dependencies in here?

Reply:
Not sure what you mean, this is a entry in our `<dependencyManagement>` block